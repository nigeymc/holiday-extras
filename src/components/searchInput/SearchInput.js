import PropTypes from 'prop-types';

const SearchInput = ({ searchTerm, handleChange, filter }) => {
    return (
        <form className="photo-stream__form">
            <div className="form__container">
                <label htmlFor="tag-search">Search Tags </label>
                <input type="text" name="tag-search" placeholder="Search photo tags" aria-label="tag-search" value={searchTerm} onChange={handleChange} filter={filter} />
            </div>
        </form>
    )
}

SearchInput.propTypes = {
    searchTerm: PropTypes.string,
    handleChange: PropTypes.func,
    filter: PropTypes.func
}

export { SearchInput as default };