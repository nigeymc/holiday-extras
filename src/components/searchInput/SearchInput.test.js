import { shallow } from 'enzyme';
import SearchInput from './SearchInput';

test('renders component correctly', () => {
    const wrapper = shallow(<SearchInput />)
    expect((wrapper)).toMatchSnapshot();
});