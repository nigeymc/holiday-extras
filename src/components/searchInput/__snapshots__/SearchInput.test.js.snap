// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component correctly 1`] = `ShallowWrapper {}`;
